examples/02-01-document-exploration.ipynb

```diff
@@ -0,0 +1,326 @@
+      "text/plain": [
+       "<Figure size 720x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "from wordcloud import WordCloud, STOPWORDS\n",
```

Comment:
Is STOPWORDS in the last cell have any hidden effect or is it imported accidentally?

Reply:
Thanks for noticing. I fixed it.